ff_locking.c

```diff
@@ -293,10 +294,11 @@ BaseType_t xReturn;
 		return pdTRUE;
 	}
 	/* This function is called when a task is waiting for a sector buffer
-	to become available. */
+	to become available.  Each time when a sector buffer becomes available,
+	the bit will be set ( see FF_BufferProceed() here below ). */
 	xBits = xEventGroupWaitBits( pxIOManager->xEventGroup,
 		FF_BUF_LOCK_EVENT_BITS, /* uxBitsToWaitFor */
-		FF_BUF_LOCK_EVENT_BITS, /* xClearOnExit */
+		pdTRUE,                 /* xClearOnExit */
```

Comment:
I don't have quite a bit of background here, but do we need this change?

Reply:
It was my bad, a long time ago: I mistakenly thought that `xClearOnExit` was a bit mask. In stead it is a boolean value :

If `xClearOnExit` is set to `pdTRUE` then any bits set in the value passed as the `uxBitsToWaitFor` parameter will be cleared in the event group before `xEventGroupWaitBits()` returns if `xEventGroupWaitBits()` returns for any reason other than a timeout.  The timeout value is set by the `xTicksToWait` parameter.
( Original text found [here](https://www.freertos.org/xEventGroupWaitBits.html) ).

So the use of `pdTRUE` is correct here.